vra/resource_machine.go

```diff
@@ -299,7 +299,33 @@ func resourceMachineUpdate(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
-	log.Printf("Finished updating the vra_machine resource with name %s", d.Get("name"))
```

Comment:
Looks good. Please add this log statement back so that when flavor is not updated, you still have a log message.